admin/src/components/SideBar.tsx

```diff
@@ -0,0 +1,33 @@
+import { FaTimes } from 'react-icons/fa';
+import { RxDashboard } from "react-icons/rx";
+import { HiOutlineUsers } from "react-icons/hi2";
+import { CgProfile } from "react-icons/cg";
+import { IoNewspaperOutline } from "react-icons/io5";
+import logo from '../assets/favicon.png';
+import { Link, useLocation } from 'react-router-dom';
+
+const Sider = ({ sidebarOpen, toggleSidebar }: { sidebarOpen: boolean, toggleSidebar: () => void }) => {
```

Comment:
Can we rename this component to be `SideBar` instead of `Sider` ? As it might be confusing for others on what does Sider means.

Reply:
@akbatra567 Done 👍